src/executables/uber.js

```diff
@@ -6,5 +6,5 @@ import pkg from '../../package.json';
 
 program.version(pkg.version)
   .command('price', 'get price estimate')
-  .command('time', 'get time to pickup estimate')
+  .command('time <address>', 'get time to pickup estimate')
```

Comment:
Hmm I think I was able to get this to work locally without the `<address>` - mind double checking?